festim/hydrogen_transport_problem.py

```diff
@@ -99,17 +99,23 @@ def __init__(
         self.formulation = None
         self.volume_subdomains = []
         self.bc_forms = []
+        self.temperature_fenics_value = None
+        self.temperature_time_dependent = False
 
     @property
-    def temperature(self):
-        return self._temperature
+    def temperature_fenics_value(self):
```

Comment:
Do we want to find something shorter like `temperature_fenics` or `temperature_value`?